rrfs-test/validated_yamls/templates/obtype_config/msonet_winds_288.yaml

```diff
@@ -0,0 +1,521 @@
+           action:
+             name: assign error
+             error function:
+               name: ObsFunction/ObsErrorModelStepwiseLinear
+               options:
+                 xvar:
+                   name: MetaData/pressure
+                 xvals: [110000, 105000, 100000, 95000, 90000, 85000, 80000, 75000, 70000, 65000, 60000, 55000, 50000, 45000, 40000, 35000, 30000, 25000, 20000, 15000, 10000, 7500, 5000, 4000, 3000, 2000, 1000, 500, 400, 300, 200, 100, 0]
+                 errors: [3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0, 3.0]
+
+         # Error inflation based on pressure check (setupw.f90)
+         - filter: Perform Action
+           filter variables:
+           - name: windEastward
+           where:
+           - variable: ObsType/windEastward
+             is_in: 288
+           action:
+             name: inflate error
+             inflation variable:
+               name: ObsFunction/ObsErrorFactorPressureCheck
+               options:
+                 variable: windEastward
+                 inflation factor: 4.0
+
+         # Error inflation based on pressure check (setupw.f90)
+         - filter: Perform Action
+           filter variables:
+           - name: windNorthward
+           where:
+           - variable: ObsType/windNorthward
+             is_in: 288
+           action:
+             name: inflate error
+             inflation variable:
+               name: ObsFunction/ObsErrorFactorPressureCheck
+               options:
+                 variable: windNorthward
+                 inflation factor: 4.0
+
+         # Bounds Check
+         - filter: Bounds Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windEastward
+           - name: windNorthward
+           minvalue: -130
+           maxvalue: 130
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           filter variables:
+           - name: windEastward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     &cgross    [ 5.0 ]
+               error_min:  &error_min [ 1.0 ]
+               error_max:  &error_max [ 5.0 ]
+               variable: windEastward
+           where:
+           - variable: QualityMarker/windEastward
+             is_not_in: 3
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           filter variables:
+           - name: windNorthward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     *cgross
+               error_min:  *error_min
+               error_max:  *error_max
```

Comment:
I'm not pretty sure if we should use a reference here, as well as for temperature, humidity, and surface pressure. It might be fine when using just one type of observations, but does the reference still hold when we combine multiple observations? Even if it does work, I think these references could lead to confusion for users since each may represent different values for different observations.

Reply:
If we want to use a reference here, it might be better to choose a name like &error_max_wind_288?

Reply:
Yeah, I think you might be right. I hadn't tried it with another wind obs space. I wasn't defining these in the ps/t/q msonet obs space, so I didn't catch the issue. I was only defining them in the uv obs space because it has both eastward and westward winds so 4 total gross checks I think I'm now leaning toward just removing the references. What is your preference @spanNOAA?

Reply:
I was using the exact numbers. I'm preparing to align with your standards and submit a draft PR, and then this question came to mind.

Reply:
@spanNOAA, I just removed the anchors and aliases.